['/home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/buildtools/linux64/gn', 'gen', '/tmp/out/Default']
ERROR at //chrome/BUILD.gn:269:20: Can't load input file.
    data_deps += [ "//third_party/widevine/cdm:widevinecdmadapter" ]
                   ^----------------------------------------------
Unable to load:
  /home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/third_party/widevine/cdm/BUILD.gn
I also checked in the secondary tree for:
  /home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/build/secondary/third_party/widevine/cdm/BUILD.gn
==> ERROR: A failure occurred in prepare().
    Aborting...


['/home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/buildtools/linux64/gn', 'gen', '/tmp/out/Default']
ERROR at //media/BUILD.gn:605:5: Can't load input file.
    "//third_party/widevine/cdm:version_h",
    ^-------------------------------------
Unable to load:
  /home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/third_party/widevine/cdm/BUILD.gn
I also checked in the secondary tree for:
  /home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/build/secondary/third_party/widevine/cdm/BUILD.gn
==> ERROR: A failure occurred in prepare().
    Aborting...


diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index b2d01f3..c6de290 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -266,7 +266,7 @@ if (!is_android && !is_mac) {
       "//third_party/adobe/flash:flapper_binaries",
     ]
 
-    data_deps += [ "//third_party/widevine/cdm:widevinecdmadapter" ]
+#    data_deps += [ "//third_party/widevine/cdm:widevinecdmadapter" ]
 
     if (is_multi_dll_chrome) {
       defines += [ "CHROME_MULTIPLE_DLL" ]
diff --git a/media/BUILD.gn b/media/BUILD.gn
index a1f03af..cbb651f 100644
--- a/media/BUILD.gn
+++ b/media/BUILD.gn
@@ -602,7 +602,7 @@ test("media_unittests") {
     "//testing/gtest",
     "//third_party/libwebm",
     "//third_party/libyuv",
-    "//third_party/widevine/cdm:version_h",
+#    "//third_party/widevine/cdm:version_h",
     "//ui/gfx:test_support",
     "//url",
   ]
@@ -725,7 +725,7 @@ test("media_perftests") {
     "//testing/gmock",
     "//testing/gtest",
     "//testing/perf",
-    "//third_party/widevine/cdm:version_h",
+#    "//third_party/widevine/cdm:version_h",
     "//ui/gfx:test_support",
   ]
   if (media_use_ffmpeg) {
